src/main/java/com/healthyfood/connectivity/FetchApiFood.java

```diff
@@ -0,0 +1,60 @@
+package com.healthyfood.connectivity;
+
+import com.healthyfood.utils.CapitalizeTheFirstLetter;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Collections;
+import java.util.List;
+
+public class FetchApiFood {
+    private final HttpClient client = HttpClient.newBuilder().build();
+
+    public String getProductByName(String productName) {
+        try {
+            HttpRequest request = HttpRequest.newBuilder().uri(URI.create("https://api.edamam.com/api/food-database/v2/parser?app_id=a5c1a6e2&app_key=%20917d9904d1aa8302eb4d353787034eb6&ingr=" + CapitalizeTheFirstLetter.capitalizeTheFirstLetter(productName) + "&nutrition-type=cooking")).GET().build();
```

Comment:
please put the URI into a constant variable

Reply:
Ok